src/main/resources/static/script.js

```diff
@@ -1,10 +1,39 @@
-let centerLatitude = 41.85;
-let centerLongitude = -87.65;
-let centerZoom = 7;
+
+var map;
+let centerLatitude = 37.85;
+let centerLongitude = -97.65;
+let centerZoom = 4;
 
 function initMap() {
+   var originInput = document.getElementById("originInput");
+   var origin = new google.maps.places.Autocomplete(originInput,
+   {
+     componentRestrictions: {'country': ['us']},
+     fields: ['geometry', 'name', 'formatted_address']
+   });
+   var destinationInput = document.getElementById("destinationInput");
+   var destination = new google.maps.places.Autocomplete(destinationInput,
+   {
+    componentRestrictions: {'country': ['us']},
+    fields: ['geometry', 'name', 'formatted_address']
+   });
+     let startInput = document.getElementById("originInput");
+            google.maps.event.addDomListener(startInput, "keydown", function(event) {
+            if (event.keyCode === 13){
+              event.preventDefault();
+              }
+            });
+       let endInput = document.getElementById("destinationInput");
+              google.maps.event.addDomListener(destinationInput, "keydown", function(event) {
+              if (event.keyCode === 13){
+                event.preventDefault();
+                }
+              });
+
```

Comment:
I have one more request! The script.js file is currently styled with two spaces per indentation, and it looks like this new code is uses 4 spaces/a full tab. can you refactor to keep the indentation congruent throughout? TYSM!

Reply:
```suggestion
changed to 2 spaces throughout
```